src/libYARP_manager/include/yarp/manager/executable.h

```diff
@@ -27,8 +27,8 @@ namespace yarp {
 namespace manager {
 
 
-#define DEF_PERIOD      100  //ms
-#define WDOG_PERIOD     5000 //ms
+#define DEF_PERIOD      0.1  //s
+#define WDOG_PERIOD     0.5 //s
```

Comment:
I think this should be `#define WDOG_PERIOD     5.0` to be consistent with the old version.